docs/src/usage.md

```diff
@@ -0,0 +1,12 @@
+# Usage
+
+At present, Kani can used in two ways:
+ * [On a single file](./kani-single-file.md) with the `kani` command.
+ * [On a package](./cargo-kani.md) with the `cargo-kani` command.
+
+Running [Kani on a single file](./kani-single-file.md) is quite useful for small examples.
+
+However, if you plan to integrate Kani in your projects, the recommended
+approach is to use [Kani on a package](./cargo-kani.md) because it includes
+utilities for tagging, extracting and running all proof harnesses in your
+package without making changes to its codebase.
```

Comment:
It's not clear to me which changes users would have to make to their codebase. I would say that the main difference today between `kani` and `cargo kani` is the ability to handle external dependencies.

Reply:
I mean, the ability to build more than a single `.rs` file at all. I don't think `kani` is even useful for customers, really.

Reply:
You are able to verify a full crate with standalone kani, independent on the number of files, as long as it doesn't have any external dependency.

For example, you can verify the rectangle example inside [tests/cargo-kani/rectangle-example](https://github.com/model-checking/kani/tree/main/tests/cargo-kani/rectangle-example) with the following command:
```
kani src/lib.rs --function stretched_rectangle_can_hold_original
```
Note that the crate has two files: `src/lib.rs` and `src/rectangle.rs`.

Reply:
You are able to verify a full crate with standalone kani, independent on the number of files, as long as it doesn't have any external dependency.

For example, you can verify the rectangle example inside [tests/cargo-kani/rectangle-example](https://github.com/model-checking/kani/tree/main/tests/cargo-kani/rectangle-example) with the following command:
```
kani src/lib.rs --function stretched_rectangle_can_hold_original
```
Note that the crate has two files: `src/lib.rs` and `src/rectangle.rs`.

Reply:
Oh neat. TIL

Reply:
> Oh neat. TIL

+1

Let's just write that then, no?
```suggestion
However, if you plan to integrate Kani in your projects, the recommended
approach is to use [Kani on a package](./cargo-kani.md) because of its ability
to handle external dependencies.
```